sw/device/tests/power_virus_systemtest.c

```diff
@@ -684,6 +684,49 @@ static void configure_pinmux_sim(void) {
   //    Channel 0 on IOR11
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIor11,
                                         kTopEarlgreyPinmuxOutselPwmAonPwm0));
+
+  if (kDeviceType == kDeviceSilicon || kDeviceType == kDeviceSimDV) {
+    // Set fast slew rate and strong drive strengh for SPI pads
+    dif_pinmux_pad_attr_t out_attr;
+    dif_pinmux_pad_attr_t in_attr = {
+        .slew_rate = 1,
+        .drive_strength = 3,
+        .flags = kDifPinmuxPadAttrPullResistorEnable |
+                 kDifPinmuxPadAttrPullResistorUp};
+
+    CHECK_DIF_OK(
+        dif_pinmux_pad_write_attrs(&pinmux, kTopEarlgreyDirectPadsSpiHost0Sck,
+                                   kDifPinmuxPadKindDio, in_attr, &out_attr));
+    CHECK_DIF_OK(
+        dif_pinmux_pad_write_attrs(&pinmux, kTopEarlgreyDirectPadsSpiHost0Sd0,
+                                   kDifPinmuxPadKindDio, in_attr, &out_attr));
+    CHECK_DIF_OK(
+        dif_pinmux_pad_write_attrs(&pinmux, kTopEarlgreyDirectPadsSpiHost0Sd1,
+                                   kDifPinmuxPadKindDio, in_attr, &out_attr));
+
+    CHECK_DIF_OK(
+        dif_pinmux_pad_write_attrs(&pinmux, kTopEarlgreyDirectPadsSpiHost0Sd2,
+                                   kDifPinmuxPadKindDio, in_attr, &out_attr));
+
+    CHECK_DIF_OK(
+        dif_pinmux_pad_write_attrs(&pinmux, kTopEarlgreyDirectPadsSpiHost0Sd3,
+                                   kDifPinmuxPadKindDio, in_attr, &out_attr));
+    // Spi_Device
```

Comment:
Aren't these attributes already set on the SPI Host 0 data pins? See lines 580 -- 601 in this file. Unless I am missing something here?

Reply:
I think lines 580-601 only set pull up and not drive strength/slew rate.

Reply:
sure, but can you merge this config with the lines I mentioned above? So we don't set the attributes twice?

Reply:
I tried, but dif_pinmux_pad_attr_flags https://github.com/lowRISC/opentitan/blob/ae8ac572e793f34f457d08c9ce03db3644c49f5a/sw/device/lib/dif/dif_pinmux.h#L140 doesn't have drive_strength and slew_rate fields. Am I missing something here?

Reply:
The slew is an attribute, not an attribute flag: https://github.com/lowRISC/opentitan/blob/ae8ac572e793f34f457d08c9ce03db3644c49f5a/sw/device/lib/dif/dif_pinmux.h#L158

Reply:
Sorry should have been more clear here. To do this properly would require an alignment between these two structs: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/pinmux_testutils.h;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=99

and

https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/dif/dif_pinmux.h;l=168?q=dif_pinmux.h&ss=opentitan%2Fopentitan

Which is more refactoring. It is ok to leave as is in this PR and we can file a follow up PR to align these later.